packages/core/merge.js

```diff
@@ -27,7 +27,7 @@ function mergeOne(dst, src) {
     // is this a regular property?
     if (desc.hasOwnProperty('value')) { // eslint-disable-line
       // Do not merge properties with the 'undefined' value.
-      if (desc.value === undefined) return;
+      if (desc.value === undefined) return dst;
```

Comment:
Everything is good with this PR. But this line.

The `merge` algorithm is probably the most difficult part of stampit. I would need to take a very deep look at it before approving the PR. But I don't have time (my newborn is taking over me).

What you can do is go step by step:
* Remove the `.js` file changes from this PR. Leave everything else you've updated.
* I'll merge this PR.
* Submit another PR with `.js` file changes for further review.

Reply:
your wish is my command